src/app/services/category.service.ts

```diff
@@ -0,0 +1,20 @@
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+import { Observable } from 'rxjs';
+import { CategoryModel } from '../models/category.model';
+
+@Injectable()
+export class CategoryService {
+  private readonly _categoryUrl: string =
```

Comment:
It can be declarated above class as const variable and than you can use it in your class